### PR TITLE
Ref #30 API for delete checkout key

### PR DESCRIPTION
**Pull request by anujaware**

### CODE REVIEW

File: lib/circleci/project.rb [hunk 1]
```diff
@@ -130,6 +130,19 @@ def self.settings username, project
...
     end
 
+    ##
+    #
+    # Delete a checkout key
+    #
+    # @param username     [String] - User or org name who owns project
+    # @param project      [String] - Name of project
+    # @param fingerprint  [String] - Fingerprint
+    # @return         [CircleCi::Response] - Response object
```
Comment (mtchavez) [inline lib/circleci/project.rb]: Correct the spacing here please to match the other indentation of parameter types.

File: README.md [hunk 2]
```diff
@@ -46,6 +46,7 @@ end
   * [Recent Builds](#recent_builds)
   * [Settings](#settings)
   * [Unfollow](#unfollow)
+  * [Delete Checkout](#delete_checkout)
```
Comment (mtchavez) [inline README.md]: These are in alphabetical order for easy grokking. Please alphabetize.

File: README.md [hunk 3]
```diff
@@ -454,6 +455,26 @@ Example response
 }
 ```
 
+#### [delete checkout](#delete_checkout)
```
Comment (mtchavez) [inline README.md]: This section needs to be alphabetized as well.

**Comment by mtchavez** [comment]
Please rebase against master, as the branch has merge conflicts. Added comments, once those are addressed this looks good. Thanks.

**Comment by anujaware** [comment]
Thanks @mtchavez . I made the changes and PL is complete. Please merge this.

**Pull request closed without merging**